test/commands/diff.test.ts

```diff
@@ -1,12 +1,16 @@
 /* eslint-disable max-len */
 import { expect, test } from '@oclif/test'
-import { AUTH_URL } from '../../src/api/common'
+import { AUTH_URL, BASE_URL } from '../../src/api/common'
+
+process.env = {}
 
 const expected = `
 DevCycle Variable Changes:
 
-✅ 4 Variables Added
-❌ 1 Variable Removed
+⚠️   0 Variables With Notices
```

Comment:
In the case that we don't have notices, I think we should omit this line

Reply:
agreed